src/controller_store.coffee

```diff
@@ -0,0 +1,28 @@
+camelCase = require 'camel-case'
+FileIdentifier = require './file_identifier'
+fs = require 'fs'
+path = require 'path'
+
+
+class ControllerStore
+
+  constructor: (@controllersDir, @dependencies) ->
+    @controllers = {}
+
+
+  # Calls a controller action with args
+  applyAction: ({controllerName, actionName, args}) ->
+    @controllers[camelCase controllerName][actionName] args...
+
+
+  # Populates @controllers by instantiating controllers found in @controllersDir
+  initialize: ->
+    for fileName in fs.readdirSync @controllersDir
```

Comment:
Yeah, this should be pulled into a separate PR since it's a bigger change. We'll want to block http requests until all controllers are initialized.

Also for most applications using exprestive this will run once as the application boots up so it shouldn't really be blocking anything important anyway

But I do agree that we should eventually do this